proxies/live/apiproxy/policies/AssignMessage.Errors.Default.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Will also need to add this to the default.xml ProxyEndPoint as the CatchallErrorMessage is used there, too

Reply:
Added to both.